core/src/main/java/org/polypheny/db/algebra/core/lpg/LpgTransformer.java

```diff
@@ -48,7 +48,8 @@ public LpgTransformer( AlgOptCluster cluster, AlgTraitSet traitSet, List<AlgNode
 
     @Override
     public String algCompareString() {
-        return "$" + getClass().getSimpleName() + "$" + getInputs().stream().map( AlgNode::algCompareString ).collect( Collectors.joining( "$" ) );
+        return getClass().getSimpleName() + "$"
+                + getInputs().stream().map( AlgNode::algCompareString ).collect( Collectors.joining( "$" ) ) + "&";
```

Comment:
operationOrder is not part of the algCompareString. Intentional?

Reply:
Added.